src/components/Download/DownloadLink.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import DownloadLink from './DownloadLink';
+
+const meta = {
+  title: 'Components/Download/DownloadLink',
```

Comment:
I can. Since this differs from the Download component [ticket](https://bsc-oit.atlassian.net/browse/ADS-795) and [Zeplin](https://app.zeplin.io/project/6075edaf528a0a46f9447bd5/screen/64513eff73de712645877472) I wasn't sure we'd be sticking with it. Will update in next push.